### PR TITLE
Feature/exit

### PR DESCRIPTION
**Pull request by andreGarvin**
@rdepena @datamadic @whyn07m3

### CODE REVIEW

File: src/api/system/system.ts [hunk 1]
```diff
@@ -80,6 +80,15 @@ export default class System extends Base {
     }
 
     /**
+     * Exits the Runtime.
+     * @return {Promise.<void>}
+     * @tutorial System.exit
+     */
+    public exit(): Promise<void> {
+        return this.wire.sendAction('exit-desktop', Object.assign({})).then(() => undefined);
```
Comment (whyn07m3) [inline src/api/system/system.ts]: • Why `Object.assign({})` here?
• Why `.then(() => undefined);` here?

File: tutorials/System.exit.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Exits the Runtime
+
+# Example
+```js
+fin.System.exit().then(() => console.log('Exiting.')).catch(err => console.log(err));
```
Comment (whyn07m3) [inline tutorials/System.exit.md]: • Maybe something like "Runtime exiting" or "Runtime exited" is clearer
• Maybe this: `.catch(console.error);`

File: src/api/system/system.ts (hunk 1 above)
Comment (andreGarvin) [reply src/api/system/system.ts]: I thought `this.wire` needed an Object, I just given is an empty Object not `Object.assign({})`

Comment (andreGarvin) [reply src/api/system/system.ts]: Also I wrote `.then(() => undefined)` becasue in the javascript-adapter it does not recevice any args/input.

**Pull request closed without merging**